pages/about/policies.html

```diff
@@ -28,7 +28,7 @@
         $('#healthsafetycontent').load("policies/healthsafety.html");
         $("#homeworkcontent").load("policies/homework.html");
         $("#criticalincidentmanagementcontent").load("policies/criticalincidentmanagement.html");
-        $("#relationshipsexualeducationcontent").load("policies/relationshipsexualeducation.html");
+        $("#relationshipsexualeducationcontent").load("policies/relationshipsexualed.html");
```

Comment:
@denispop9 what's the reason behind changing the file name?

Reply:
the file name was named 'relationshipsexulaed.html' so it wouldn't pop up when I pressed 'read me'.